website/other/bezier-rs-demos/src/features/bezier-features.ts

```diff
@@ -111,54 +111,58 @@ const bezierFeatures = {
 		},
 	},
 	Tangent: {
-		callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.tangent(options.t),
+		callback: (bezier: WasmBezierInstance, options: Record<string, number>, _: undefined, computeType: ComputeType): string => bezier.tangent(options.c, computeType),
 		demoOptions: {
 			Quadratic: {
-				sliderOptions: [tSliderOptions],
+				sliderOptions: [cSliderOptions],
 			},
 		},
+		chooseComputeType: true,
 	},
 	Normal: {
-		callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.normal(options.t),
+		callback: (bezier: WasmBezierInstance, options: Record<string, number>, _: undefined, computeType: ComputeType): string => bezier.normal(options.c, computeType),
 		demoOptions: {
 			Quadratic: {
-				sliderOptions: [tSliderOptions],
+				sliderOptions: [cSliderOptions],
 			},
 		},
+		chooseComputeType: true,
 	},
 	Curvature: {
-		callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.curvature(options.t),
+		callback: (bezier: WasmBezierInstance, options: Record<string, number>, _: undefined, computeType: ComputeType): string => bezier.curvature(options.c, computeType),
 		demoOptions: {
 			Linear: {
 				disabled: true,
 			},
 			Quadratic: {
-				sliderOptions: [tSliderOptions],
+				sliderOptions: [cSliderOptions],
 			},
 		},
+		chooseComputeType: true,
 	},
 	Split: {
-		callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.split(options.t),
+		callback: (bezier: WasmBezierInstance, options: Record<string, number>, _: undefined, computeType: ComputeType): string => bezier.split(options.c, computeType),
 		demoOptions: {
 			Quadratic: {
-				sliderOptions: [tSliderOptions],
+				sliderOptions: [cSliderOptions],
 			},
 		},
+		chooseComputeType: true,
 	},
 	Trim: {
-		callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.trim(options.t1, options.t2),
+		callback: (bezier: WasmBezierInstance, options: Record<string, number>, _: undefined, computeType: ComputeType): string => bezier.trim(options.c2, options.c1, computeType),
```

Comment:
Is there a reason why `c2` is passed before `c1`?

Reply:
good catch